WebAPIDemo1/Service/BookService.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using WebAPIDemo1.Database;
+using WebAPIDemo1.Service;
+
+namespace WebAPIDemo1.Model
+{
```

Comment:
Overall issues in method signature, for instance add should return back added book data.
In case of invalid data we should inform user what information is incorrect.
Say if Name is invalid we should tell user that the name is invalid.